x/bsonx/bsoncore/bsonbuilder.go

```diff
@@ -0,0 +1,130 @@
+// Copyright (C) MongoDB, Inc. 2017-present.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License. You may obtain
+// a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+// Package bsoncore contains functions that can be used to encode and decode bson
```

Comment:
We don't need this package-level documentation in every file. It should really only be in `bsoncore/doc.go`. I'm not sure if that actually exists, but you can remove this block comment either way.